packages/graph/hash_graph/lib/graph/src/store/postgres/query/expression/conditional.rs

```diff
@@ -35,6 +35,7 @@ impl Transpile for Function<'_> {
 /// A compiled expression in Postgres.
 #[derive(Debug, PartialEq, Eq, Hash)]
 pub enum Expression<'q> {
+    Asterix,
```

Comment:
I believe it would be this instead :)
```suggestion
    Asterisk,
```